app/assets/stylesheets/application.css.scss

```diff
@@ -107,6 +105,24 @@ td, th {
   }
 }
 
+.card .card-action a {
+  color:black;
+}
+.card .card-action a:hover {
+  color: green;
+}
+
+.btn:hover {
+  background-color: #00b8d4;
```

Comment:
Color literals like `#00b8d4` should only be used in variable declarations; they should be referred to via variable everywhere else.